src/sentry/api/serializers/models/organization_member/response.py

```diff
@@ -80,5 +81,6 @@ class OrganizationMemberWithProjectsResponse(OrganizationMemberResponse):
 class OrganizationMemberWithRolesResponse(OrganizationMemberWithTeamsResponse):
     invite_link: Optional[str]
     isOnlyOwner: bool
-    roles: List[RoleSerializerResponse]
-    teamRoles: List[RoleSerializerResponse]
+    roles: List[RoleSerializerResponse]  # Deprecating: use orgRoleList
+    orgRoleList: List[RoleSerializerResponse]
+    teamRoleList: List[RoleSerializerResponse]
```

Comment:
Would `orgRoles` and `teamRoles` work here? Having `List` in the name seems a bit redundant given that the property is also a list.

Reply:
The issue with naming it as `teamRoles` is that it can be confused for being the list of options, or the list of team ⇋ member with roles for each team. 

IMO calling it `teamRoleList` or `teamRoleOptions` makes it clear. Open to other descriptive terms too.

Reply:
`teamRoleMenu`?